services/processor/processor/handlers/create_project.py

```diff
@@ -0,0 +1,125 @@
+"""
```

Comment:
I would rename the file to `project_sync.py` as `create_project` is more "actinable" name.